include/avif/avif.h

```diff
@@ -350,9 +353,10 @@ enum
     AVIF_TRANSFER_CHARACTERISTICS_SRGB = 13,
     AVIF_TRANSFER_CHARACTERISTICS_BT2020_10BIT = 14,
     AVIF_TRANSFER_CHARACTERISTICS_BT2020_12BIT = 15,
-    AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 = 16, // Perceptual Quantizer; BT.2100 PQ
+    AVIF_TRANSFER_CHARACTERISTICS_PQ = 16, // Perceptual Quantizer (HDR); BT.2100 PQ
+    AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 = 16,
```

Comment:
> Optional: It may be good to still list the original `AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084` first to suggest that `AVIF_TRANSFER_CHARACTERISTICS_PQ` is an alias. But we may make an exception here because most people refer to this as PQ.

I think people are more familiar with sRGB and PQ in general so I thought putting them first was clearer. I am not sure what order you favor though so feel free to send a PR.

> Note: The enum names used in avif.h are based on the names used in the AV1 specification to make it easier to work with the AV1 codecs.

Yes I kept some in the code base, depending on the nearby comments.